dist/map-panel.js

```diff
@@ -130,7 +130,6 @@ Fliplet.InteractiveMap.component('map-panel', {
     openMapPicker: function openMapPicker() {
       var _this = this;
 
-      Fliplet.Widget.toggleCancelButton(false);
```

Comment:
why are these gone away?

Reply:
@squallstar This happened because @YaroslavOvdii made PR and changed only dist files. We discussed this problem with him after this PR will be merged, he will update his PR using gulp.

Reply:
@MaksymShokin great, please link the PR to a follow up comment so I can resolve this comment

Reply:
@MaksymShokin if your editor support ESLint, this repository is set up with a file to automatically fix this for you and warn you about problems: https://github.com/Fliplet/fliplet-widget-interactive-map/blob/master/.eslintrc